lib/LedgerSMB/DBObject/Reconciliation.pm

```diff
@@ -162,7 +162,9 @@ sub approve {
     # the user should be embedded into the $self object.
     my $report_id = shift @_;
 
-    my $code = $self->call_dbmethod(funcname=>'reconciliation__report_approve', args=>[$report_id]); # user
+    # Even with https://github.com/ledgersmb/LedgerSMB/issues/838 fixes, array parameters DO NOT WORK
+    #my $code = $self->call_dbmethod(funcname=>'reconciliation__report_approve', args=>[$report_id]); # user
+    my $code = $self->call_dbmethod(funcname=>'reconciliation__report_approve', args=> { in_report_id => $report_id }); # user
```

Comment:
I was about to, but call_procedure is using positional parameters and I have a problem with that.